.circleci/config.yml

```diff
@@ -1,220 +1,72 @@
-      - run: npx snyk monitor --org=customer-products
-          --project-name=Financial-Times/n-express
-      - run:
-          name: shared-helper / npm-version-and-publish-public
-          command: .circleci/shared-helpers/helper-npm-version-and-publish-public
-
+      - tool-kit/persist-workspace:
+          path: .
 workflows:
-
   version: 2
-
-  build-test:
+  tool-kit:
     when:
       not:
         equal:
           - scheduled_pipeline
           - << pipeline.trigger_source >>
     jobs:
-      - build:
+      - checkout:
           filters:
-            <<: *filters_ignore_tags_renovate_nori
-          name: build-v<< matrix.node-version >>
-          matrix:
-            parameters:
-              node-version: [ "16.14", "14.19" ]
-      - test:
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
+      - waiting-for-approval:
+          type: approval
+          filters:
+            branches:
+              only: /(^renovate-.*|^nori/.*)/
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
+      - tool-kit/setup:
           requires:
-            - build-v<< matrix.node-version >>
-          name: test-v<< matrix.node-version >>
-          matrix:
-            parameters:
-              node-version: [ "16.14", "14.19" ]
-
-  build-test-publish:
-    when:
-      not:
-        equal:
-          - scheduled_pipeline
-          - << pipeline.trigger_source >>
-    jobs:
-      - build:
+            - checkout
+            - waiting-for-approval
           filters:
-            <<: *filters_version_tag
-          name: build-v<< matrix.node-version >>
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
+      - tool-kit/build:
+          requires:
+            - tool-kit/setup
+          name: tool-kit/build-v<< matrix.node-version >>
           matrix:
             parameters:
               node-version: [ "16.14", "14.19" ]
-      - test:
           filters:
-            <<: *filters_version_tag
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
+      - tool-kit/test:
           requires:
-            - build-v<< matrix.node-version >>
-          name: test-v<< matrix.node-version >>
+            - tool-kit/build-v<< matrix.node-version >>
+          name: tool-kit/test-v<< matrix.node-version >>
           matrix:
             parameters:
               node-version: [ "16.14", "14.19" ]
-      - publish:
-          context: npm-publish-token
           filters:
-            <<: *filters_version_tag
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
+      - tool-kit/publish:
+          node-version: 16.14-browsers
+          context: npm-publish-token
           requires:
-            - test-v16.14
-
-  renovate-nori-build-test:
-    when:
-      not:
-        equal:
-          - scheduled_pipeline
-          - << pipeline.trigger_source >>
-    jobs:
-      - waiting-for-approval:
-          type: approval
+            - tool-kit/test
```

Comment:
**issue**: the `tool-kit/test` jobs' names have been appended by the Node version you're using so you need to include the full names here

You could do either:
```suggestion
            - tool-kit/test-16.14
```
which is what the migration script would have generated previously, or:
```suggestion
            - tool-kit/test-16.14
            - tool-kit/test-14.19
```
which would only publish the package if it passes for Node 14 as well. the latter makes more sense to me in retrospect though maybe there was a reason I didn't implement it like that at the time...